### PR TITLE
Introduced Codable

### PR DESCRIPTION
**Pull request by theapache64**

### CODE REVIEW

**Comment by theapache64** [comment]
Please respond !

**Comment by mkko** [comment]
Hi, thanks for the addition and apologies for not taking this sooner. I would still keep the JSON parsing libraries mentioned, as they might be relevant to someone.

**Comment by mstuhlbein** [comment]
I would also encourage the link to point to the [official documentation](https://developer.apple.com/documentation/foundation/archives_and_serialization/encoding_and_decoding_custom_types).

**Comment by mkko** [comment]
> I would also encourage the link to point to the [official documentation](https://developer.apple.com/documentation/foundation/archives_and_serialization/encoding_and_decoding_custom_types).

Agree. @theapache64, do you mind making the changes?

**Comment by theapache64** [comment]
yeah sure. will update soon (y).

File: README.md [hunk 1]
```diff
@@ -259,10 +259,11 @@ These are the idiomatic ways for components to notify others about things:
 
 ### Models
 
-Keep your models immutable, and use them to translate the remote API's semantics and types to your app. For Objective-C projects, Github's [Mantle](https://github.com/Mantle/Mantle) is a good choice. In Swift, you can use structs instead of classes to ensure immutability, and use a parsing library such as [SwiftyJSON][swiftyjson] or [Argo][argo] to do the JSON-to-model mapping.
+Keep your models immutable, and use them to translate the remote API's semantics and types to your app. For Objective-C projects, Github's [Mantle](https://github.com/Mantle/Mantle) is a good choice. In Swift, you can use structs instead of classes to ensure immutability, and use Swift's [Codable][codableLink] to do the JSON-to-model mapping. There are also few third party libraries available. [SwiftyJSON][swiftyjson] and [Argo][argo] are the best among them.
```
Comment (mkko) [inline README.md]: Just one last detail: best can be highly subjective and depends heavily on the use, maybe change it to "recommended", "well-known", "popular" or something else?

Comment (theapache64) [reply README.md]: (y)

**Comment by mkko** [comment]
Merged. Thank you for your contribution @theapache64! 🎉

**Pull request merged**